sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -3943,7 +3943,7 @@ goog.scope(function() {
                                                         sFilter, sFilter, 0.0 /* lod threshold */, false /* non-normalized coords */);
             /** @type {boolean} */ var srcIsSRGB = src.getFormat().order == tcuTexture.ChannelOrder.sRGB || src.getFormat().order == tcuTexture.ChannelOrder.sRGBA;
             /** @type {boolean} */ var dstIsSRGB = dst.getFormat().order == tcuTexture.ChannelOrder.sRGB || dst.getFormat().order == tcuTexture.ChannelOrder.sRGBA;
-            /** @type {boolean} */ var convertSRGB = false;
+            /** @type {boolean} */ var convertSRGB = this.m_sRGBUpdateEnabled;
```

Comment:
See the native code: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/simplereference/sglrReferenceContext.cpp#3046. 
glu::isContextTypeES(getType()) is not defined in JS implementation, I suppose that WebGL is based on ES context. The native code to define glu::isContextTypeES() in deqp is here: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/gluRenderContext.hpp#185